### ISSUE TITLE
New user for ngeo

### ISSUE THREAD

**Issue by bvpelt** [camptocamp/ngeo#1837]
After searching for a nice library using angularjs and openlayers 3 I found ngeo. This seems to give me a lot of functionallity.

But as a new user of ngeo I need a bit more guidance to get started than the examples.

Is there some documentation on user level?
What is the difference between the ngeo and the gmf namespace?

**Comment by jusabatier** [comment]
I agree that it's very difficult to use ngeo in a project, because the documentation isn't accessible for new users...

I followed some tutorials for AngularJS, and after tried to implements ngeo's createfeature directive, but can't success.

Just for implement a map, I use `make dist` comand for generate some ngeo js files, but can't have a map with ngeo.js (minified), while it works with ngeo-debug.js

It could be usefull to have some tutorials for developping a simple map with basic functions, and moreover know which js files we need to include in our project.

PS : I tried to suscribe to ngeo-discuss on google-groups, but not yet accepted, so can't get any help to understand how this lib works.
